tests/test_sdr.py

```diff
@@ -240,8 +240,8 @@ def test_non_square_dem(self):
         expected_results = {
             'sed_export': 0.08896198869,
             'usle_tot': 1.86480903625,
-            'avoid_exp': 9204.283203125,
-            'avoid_eros': 194613.28125,
+            'avoid_exp': 9203.955078125,

```

Comment:
Total avoided erosion and avoided export values are slightly reduced because the values on stream pixels are no longer included.